test/e2e/kubernetes/node/node.go

```diff
@@ -74,14 +74,19 @@ type List struct {
 // AreAllReady returns a bool depending on cluster state
 func AreAllReady(nodeCount int) bool {
 	list, _ := Get()
+	var ready int
 	if list != nil && len(list.Nodes) == nodeCount {
 		for _, node := range list.Nodes {
 			for _, condition := range node.Status.Conditions {
-				if condition.Type == "KubeletReady" && condition.Status == "false" {
+				if condition.Type == "Ready" && condition.Status == "True" {
+					ready++
+				} else {
```

Comment:
aren't there going to be other conditions? I think this should be `if condition.Type == "Ready" && condition.Status == "False" {`